spec/factories/subjects.rb

```diff
@@ -16,7 +16,6 @@
     type {
       %w[PrimarySubject
          SecondarySubject
-         ModernLanguagesSubject
```

Comment:
why was that removed?

Reply:
Exactly, wasn't sure if we were doing after builds etc. anymore. It triggers a validation as it does not create a modern language subject which is required.

Reply:
Have noted this to look at in a follow up card.